src/plugins/data/public/query/timefilter/timefilter.ts

```diff
@@ -161,7 +170,10 @@ export class Timefilter {
     ) {
       this.refreshIntervalUpdate$.next();
       if (!newRefreshInterval.pause && newRefreshInterval.value !== 0) {
+        this.nowProvider.useSystemNow(); // turn off time syncing when auto refresh is enabled
```

Comment:
This code is bit clunky, but this is the most straightforward way I found how to use real `now` when auto refresh is enabled considering that *Send to background* is not available when auto refresh is enabled. 
The main problem is that `session.start()` isn't called on dashboard when `autoRefreshFetch$` is fired, because embeddable subscribe to it on their own. to fix that we need to make a single subscription on a dashboard level. 

I am open to opening and issue and following up on it. Then I could remove this ugly API. but I wouldn't make it in scope of this pr

Reply:
+1 for having dashboard subscribe to autorefresh and embeddables not knowing about it at all (just getting new input)

Reply:
@ppisljar removed this clunky piece and moved subscribe to auto refresh to container level